### PR TITLE
Add artifact hub file for verification

### PR DESCRIPTION
**Pull request by bruth**

### CODE REVIEW

**Review by wallyqs — APPROVED** [review]
LGTM

File: helm/charts/artifacthub-repo.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# last time it was processed. Depending on the repository kind, this is checked
+# in a different way. For Helm http based repositories, we consider it has
+# changed if the `index.yaml` file changes. For git based repositories, it does
+# when the hash of the last commit in the branch you set up changes. This does
+# NOT apply to ownership claim operations, which are processed immediately.
+#
+repositoryID: 14c17493-c338-4acf-a923-badee42eb5a0
+owners:
+  - name: Caleb Lloyd
+    email: caleb@nats.io
```
Comment (caleblloyd) [inline helm/charts/artifacthub-repo.yml:15]: ArtifactHub has sign-in with Google.  I think to claim this repo it's got to be my `synadia` address.  You might as well add yourself there too @bruth 

```suggestion
    email: caleb@synadia.com
```

Comment (bruth) [reply helm/charts/artifacthub-repo.yml:15]: @wallyqs Do you have a login?

Comment (wallyqs) [reply helm/charts/artifacthub-repo.yml:15]: have a login now using `wally@synadia.com`

**Comment by caleblloyd** [comment]
Added to the `gh-pages` branch in https://github.com/nats-io/k8s/commit/2ac8a12aa760d4ea779a6404aacc16c59458a96d

**Pull request closed without merging**